textdb/textdb-api/src/main/java/edu/uci/ics/textdb/api/common/FieldType.java

```diff
@@ -4,5 +4,6 @@
     STRING,
     INTEGER,
     DOUBLE,
-    DATE;
+    DATE,
+    TEXT;

```

Comment:
it's better to have some comments here to specify the difference between `STRING` and `TEXT`.